### PR TITLE
Merge main

### PR DESCRIPTION
**Pull request by JacksonKearl**
- improve milestone logic to not assign things to recovery milestones
- improve PR chat rendering/formatting

### CODE REVIEW

File: api/octokit.ts [hunk 1]
```diff
@@ -213,10 +213,14 @@ export class OctoKit implements GitHub {
...
-		const possibleMilestones = allMilestones.filter(
-			(milestone) =>
-				new Date(milestone.due_on) > currentDate && currentDate > new Date(milestone.created_at),
-		);
+		const possibleMilestones = allMilestones
+			.filter(
+				(milestone) =>
+					new Date(milestone.due_on) > currentDate &&
+					currentDate > new Date(milestone.created_at) &&
+					!milestone.title.includes('Recovery'),
```
Comment (lramos15) [inline api/octokit.ts:221]: Do we ever have lowercase Milestone? Not sure if we have to be careful of casing.

File: code-review-chat/CodeReviewChat.ts [hunk 2]
```diff
@@ -51,6 +52,7 @@ export class BuildChat {
 			text: message,
 			link_names: true,
 			channel: codereviewChannel.id,
+			as_user: true,
```
Comment (lramos15) [inline code-review-chat/CodeReviewChat.ts:55]: This flag is often very frowned upon in slack apps and I don't think we need it. We're using a bot token and therefore all messages will be posted as the bot and will just include the PR author's name in the message. We don't actually want to post as a user themselves https://api.slack.com/methods/chat.postMessage#authorship

**Review by lramos15 — APPROVED** [review]

File: api/octokit.ts (hunk 1 above)
Comment (JacksonKearl) [reply api/octokit.ts:221]: I don't think so, quick skim suggests it has never happened before

File: code-review-chat/CodeReviewChat.ts (hunk 2 above)
Comment (JacksonKearl) [reply code-review-chat/CodeReviewChat.ts:55]: As mentioned in PM, posting without `as_user` set causes the message to be attributed to "bot" with no icon, which isn't great. As we aren't actually providing a user name or icon override, this just allows the message to get the bot's name and icon

**Pull request merged**